lte/gateway/python/magma/enodebd/device_config/configuration_init.py

```diff
@@ -332,6 +351,133 @@ def _set_tdd_subframe_config(
     )
 
 
+def _set_neighbor_cell_list(
+        cfg: EnodebConfiguration,
+        neighbor_cell_list: Any,
+) -> None:
+    """
+    Set the LTE Neighbor cell Params as following parameters:
+     - cfg
+     - neighbor_cell_list
+     - plmn
+     - cell_id
+     - earfcn
+     - pci
+     - tac
+     - q_offset
+     - cio
+    """
+    if neighbor_cell_list and len(neighbor_cell_list) >= 1:
+        for neighbor_cell_x in neighbor_cell_list.values():
+            if neighbor_cell_x.enable:
+                i = neighbor_cell_x.index
+                desired_object_name = BaicellsParameterName.NEIGHBOR_CELL_LIST_N % i
+                cfg.add_object(desired_object_name)
+                if neighbor_cell_x.plmn:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_PLMN_N % i, neighbor_cell_x.plmn,
+                        desired_object_name,
+                    )
+                if neighbor_cell_x.cell_id:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_CELL_ID_N % i,
+                        neighbor_cell_x.cell_id,
+                        desired_object_name,
+                    )
+                if neighbor_cell_x.earfcn:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_EARFCN_N % i,
+                        neighbor_cell_x.earfcn,
+                        desired_object_name,
+                    )
+                if neighbor_cell_x.pci:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_PCI_N % i, neighbor_cell_x.pci,
+                        desired_object_name,
+                    )
+                if neighbor_cell_x.tac:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_TAC_N % i, neighbor_cell_x.tac,
+                        desired_object_name,
+                    )
+                if neighbor_cell_x.q_offset:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_QOFFSET_N % i,
+                        neighbor_cell_x.q_offset,
+                        desired_object_name,
+                    )
+                if neighbor_cell_x.cio:
+                    cfg.set_parameter_for_object(
+                        BaicellsParameterName.NEIGHBOR_CELL_CIO_N % i, neighbor_cell_x.cio,
+                        desired_object_name,
+                    )
+
+
+def _set_neighbor_freq_list(
+        cfg: EnodebConfiguration,
+        neighbor_freq_list: Any,
```

Comment:
Could you specify the type for `neighbor_freq_list`?

Reply:
Yes, I will modify the type Any to pb2.Neighborfreqlist type.